### PR TITLE
Line break and white space support in empty state

### LINKED ISSUE #765: [Enhancement] Support linebreak in text
**Is your enhancement request related to a problem? Please describe.**
In some usecases we have more then one text that we want to put into our alerts. Fx 2 texts coming from different backends. Here we want to be able to line break / seperate those text, because they are supposed to be seperated from each other. 

Like this: https://zpl.io/bl6pyB0 or https://zpl.io/bP18ExA or https://zpl.io/29zNJWA

**Describe the solution you'd like**
Maybe the alert can translate a /n to a linebreak so that we are able to do something like that?

### PR DESCRIPTION
**Pull request by SilverGull**
Closes #765 
Support for line breaks and spaces in messages in empty state and alerts

### CODE REVIEW

File: src/app/showcase/alert-showcase/alert-showcase.component.ts [hunk 1]
```diff
@@ -18,7 +18,8 @@ export class AlertShowcaseComponent {
     },
     {
       name: 'message',
-      description: '(Optional) The message shown under the title (or icon if specified)',
+      description:
+        "(Optional) The message shown under the title (or icon if specified). Use '\\n' for newline",
```
Comment (jakobe) [inline src/app/showcase/alert-showcase/alert-showcase.component.ts]: Isn't it only 1 backslash?
```suggestion
        "(Optional) The message shown under the title (or icon if specified). Use '\n' for newline",
```

File: src/app/showcase/empty-state-showcase/empty-state-showcase.component.ts [hunk 2]
```diff
@@ -27,7 +27,7 @@ export class EmptyStateShowcaseComponent {
     },
     {
       name: 'subtitle',
-      description: 'The text beneath the title.',
+      description: "The text beneath the title. Use '\\n' for newline",
```
Comment (jakobe) [inline src/app/showcase/empty-state-showcase/empty-state-showcase.component.ts]: Isn't it only 1 backslash?
```suggestion
      description: "The text beneath the title. Use '\n' for newline",
```

File: src/app/showcase/alert-showcase/alert-showcase.component.ts (hunk 1 above)
Comment (SilverGull) [reply src/app/showcase/alert-showcase/alert-showcase.component.ts]: the first one escapes the second one

File: src/app/showcase/empty-state-showcase/empty-state-showcase.component.ts (hunk 2 above)
Comment (SilverGull) [reply src/app/showcase/empty-state-showcase/empty-state-showcase.component.ts]: the first one escapes the second one :-)

**Pull request merged**